mathics/core/list.py

```diff
@@ -19,30 +19,48 @@ class ListExpression(Expression):
 
     Keyword Arguments:
         - element_properties -- properties of the collection of elements
+        - literal_values -- if this is not None, then it is a tuple of Python values
     """
 
     def __init__(
         self,
         *elements,
         elements_properties: Optional[ElementsProperties] = None,
-        is_literal: Optional[bool] = False,
+        literal_values: Optional[tuple] = None,
     ):
         self.options = None
         self.pattern_sequence = False
         self._head = SymbolList
 
+        if literal_values is not None:
+            import inspect
+
+            curframe = inspect.currentframe()
+            call_frame = inspect.getouterframes(curframe, 2)
+            print("caller name:", call_frame[1][3])
```

Comment:
Should it be kept?

Reply:
I inadvertently did not remove it. However I think it might be commented out. It was useful in debugging. And it might be useful in the future periodically, to uncomment this.

Reply:
This change is now in master e5439716